doc/source/index.rst

```diff
@@ -233,134 +228,21 @@ You can use ``ws = Workspace.from_config()`` after downloading the config file f
-      ct = ws.compute_targets[ct_name]
-
-If your compute target already exists you can call ``ct = ws.compute_targets[ct_name]``.
-
-Setting up vnet
-"""""""""""""""
-If you do not have a virtual network yet there are two ways to create one.
-
-1. Using `https://portal.azure.com <https://portal.azure.com>`_:
-
-   a. On the home page click on `+ Create a resource` on the top-left portion of the page.
-   b. Search for `Virtual Network`.
-   c. Click on `Create` and follow the instructions: select the `Subscription` and the 
-      `Resource group` you will create the vnet in, and provide a `Name` and the `Location`. 
-      **NOTE:** keep the location the same as your AzureML Workspace e.g. if your AzureML Workspace 
-      location is `eastus` you should create the Virtual Network in `East US`.
-   d. Click `Review+Create` and follow through with the rest of the instructions.
-
-2. Using Azure CLI:
-
-   a. Install the AzureCLI; instructions are here: 
-      `https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest 
-      <https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest>`_
-   b. Open terminal and login to your Azure Subscription: ``az login``. This should 
-      automatically log you in into your Azure subscription. **NOTE:** If you have more than 
-      one subscription you will need to set the right subscription to use:
-      ``az account set --subscription "<name-of-sub-to-use>"``
-   c. Create the virtual network: 
-      ``az network vnet create -g <resource-group> -n <vnet-name> --location <location> --subnet-name default``
-
-Define Environment
-~~~~~~~~~~~~~~~~~~
-
-For the `Environment <https://docs.microsoft.com/python/api/azureml-core/azureml.core.environment.environment?view=azure-ml-py>`_  
-we will use a AzureML curated environment for running Dask CPU cluster, the `AzureML-Dask-CPU`. 
-However, the ``Environment`` class allows you to specify your own docker image and additional packages to install.
-
-.. code-block:: python
-
-   env = ws.environments[env_name]
-
 Create cluster
 ~~~~~~~~~~~~~~
 
 To create cluster:
 
 .. code-block:: python
-
-   amlcluster = AzureMLCluster(
-      workspace=ws
-      , compute_target=ct
-      , environment_definition=env
-      , initial_node_count=2
-      , admin_username=admin_username
-      , admin_ssh_key=admin_ssh_key_priv   ### path, not contents of the key
-   )
+   # common optional kwargs
+   vm_size = "STANDARD_DS13_V2"                       # Azure VM size for the Compute Target
+   datastores = ws.datastores.values()                # Azure ML Datastores to mount on the headnode
+   environment = ws.environments['AzureML-Dask-CPU]   # Azure ML Environment to run on the cluster_ar
+   initial_node_count = 20                            # number of nodes to start
+   scheduler_idle_timeout = 7200                      # scheduler idle timeout in seconds 
+
+   # only workspace is required
+   amlcluster = AzureMLCluster(ws)
```

Comment:
Could we leave this with the Python example, but maybe show the optional arguments clearly. Makes copy/paste easier for users.

```suggestion
   amlcluster = AzureMLCluster(
       # Required kwargs
       workspace=ws,
       
       # Optional kwargs
       vm_size="STANDARD_DS13_V2",                        # Azure VM size for the Compute Target
       datastores=ws.datastores.values(),                 # Azure ML Datastores to mount on the headnode
       environment=ws.environments['AzureML-Dask-CPU],    # Azure ML Environment to run on the cluster_ar
       initial_node_count=20,                             # number of nodes to start
       scheduler_idle_timeout 7200                        # scheduler idle timeout in seconds 
    )
```